torch_geometric/nn/conv/gin_conv.py

```diff
@@ -141,11 +142,19 @@ def __init__(self, nn: Callable, eps: float = 0., train_eps: bool = False,
         else:
             self.register_buffer('eps', torch.Tensor([eps]))
         if edge_dim is not None:
-            if hasattr(self.nn[0], 'in_features'):
-                in_channels = self.nn[0].in_features
+            if (isinstance(self.nn, torch.nn.Sequential)
+                    or isinstance(self.nn, torch.nn.ModuleList)):
```

Comment:
I am pretty sure a ModuleList wouldn‘t work here.

Reply:
You are very right. Thanks. Fixed